File: Eraser2/app/src/main/java/com/luckly/eraser/ui/adapter/list/EraserAdapter.java
package com.luckly.eraser.ui.adapter.list;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.luckly.eraser.R;
import com.luckly.eraser.data.write.Write;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.List;

public class EraserAdapter extends RecyclerView.Adapter<EraserAdapter.EraserViewHolder> {
    List<Write> write_data;
    Context context;
    public EraserAdapter(List<Write> write_data, Context context){
            this.write_data = write_data;
            this.context = context;
    }


    @NonNull
    @Override
    public EraserViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View root = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_fragment, parent, false);
        return new EraserViewHolder(root);
    }

    @Override
    public void onBindViewHolder(@NonNull EraserViewHolder holder, int position) {

        Write write = write_data.get(position);

        SimpleDateFormat transFormat = new SimpleDateFormat("yyyy년 MM월 dd일");
        try {
            Date was = transFormat.parse(write.getDate());
            Date now = new Date(System.currentTimeMillis());
            if(now.before(was)){
                write.setTitle("당신이 지워버린 오늘의 고민");
            }else {
                write.setTitle("당신이 지워버린 과거의 고민");
            }
        } catch (ParseException e) {
            e.printStackTrace();
        }


        holder.textView_title.setText(write.getTitle());
        holder.textView_content.setText(write.getContent());
        holder.textView_date.setText(write.getDate());
    }

    @Override
    public int getItemCount() {
        return write_data != null ? write_data.size() : 0 ;
    }

    class EraserViewHolder extends RecyclerView.ViewHolder{
        TextView textView_title;
        TextView textView_date;
        TextView textView_content;

        public EraserViewHolder(@NonNull View itemView){
            super(itemView);

            textView_title = itemView.findViewById(R.id.textView_title);
            textView_content = itemView.findViewById(R.id.textView_content);
            textView_date = itemView.findViewById(R.id.textView_date);

        }
    }
}

File: Eraser2/app/src/main/java/com/luckly/eraser/ui/adapter/list/deleted/EraserAdapter.kt
package com.luckly.eraser.ui.adapter.list.deleted

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.luckly.eraser.R
import com.luckly.eraser.data.write.Write
import com.luckly.eraser.ui.adapter.list.deleted.EraserAdapter.EraserViewHolder
import java.text.ParseException
import java.text.SimpleDateFormat
import java.util.*

class EraserAdapter(var write_data: List<Write>?, var context: Context?) : RecyclerView.Adapter<EraserViewHolder>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): EraserViewHolder {
        val root = LayoutInflater.from(parent.context).inflate(R.layout.item_fragment, parent, false)
        return EraserViewHolder(root)
    }

    override fun onBindViewHolder(holder: EraserViewHolder, position: Int) {
        val write = write_data!![position]
        val transFormat = SimpleDateFormat("yyyy년 MM월 dd일")
        try {
            val was = transFormat.parse(write.Date)
            val now = Date(System.currentTimeMillis())
            if (now.before(was)) {
                write.Title = "당신이 지워버린 오늘의 고민"
            } else {
                write.Title = "당신이 지워버린 과거의 고민"
            }
        } catch (e: ParseException) {
            e.printStackTrace()
        }
        holder.textView_title.text = write.Title
        holder.textView_content.text = write.Content
        holder.textView_date.text = write.Date
    }

    override fun getItemCount(): Int {
        return if (write_data != null) write_data!!.size else 0
    }

    inner class EraserViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
        var textView_title: TextView
        var textView_date: TextView
        var textView_content: TextView

        init {
            textView_title = itemView.findViewById(R.id.textView_title)
            textView_content = itemView.findViewById(R.id.textView_content)
            textView_date = itemView.findViewById(R.id.textView_date)
        }
    }
}